src/components/list.tsx

```diff
@@ -0,0 +1,64 @@
+/* import React from 'react'; */
+import React from 'react';
+import { Crypto } from './interfaces/index';
+import {
+  Container,
+  ContainerLogo,
+  CryptoLogo,
+  ContainerText,
+  TextCoin,
+  TextSymbol,
+  ContainerLeft,
+  ContainerRight,
+  ContainerChange,
+  Arrow,
+  TextPercent,
+  TextGreen,
+  TextRed,
+  Line,
+
+} from './styles/stylesList';
+import greenarrow from './img/greenarrow.png';
+import redarrow from './img/redarrow.png';
+
+const List = ({ item }: { item: Crypto }) => (
+  <>
+    <Container>
+      <ContainerLeft>
+        <ContainerLogo>
+          <CryptoLogo source={item.img} />
+        </ContainerLogo>
+
+        <ContainerText>
+          <TextCoin>{item.name}</TextCoin>
+          <TextSymbol>{item.symbol}</TextSymbol>
+        </ContainerText>
+      </ContainerLeft>
+      <ContainerRight>
+        <ContainerChange>
+          <TextCoin>${item.price}</TextCoin>
+
+          <TextPercent>
+            {item.percent > 0 ? (
```

Comment:
redundant validation: 
you shoud use one component for this:
```
                <Arrow source={item.percent > 0  ? greenarrow  : redarrow } />
                <TextPercentage positive={item.percent > 0}> {item.percent}% </TextPercentage>
 ```
 
 Styled-components can take props, so you can validate the color dynamically inside the style